### PR TITLE
Updated Website & Download Section

### PR DESCRIPTION
**Pull request by SGNight**
> Update Download section
> Removed Js Terminal
> Update Bliss OS version information
> Few Bug Fixed

### CODE REVIEW

**Comment by hmtheboy154** [comment]
LGTM, we will temporary update the content until a new redesign coming

@electrikjesus

**Comment by electrikjesus** [comment]
Is this good to merge then?

File: index.html [hunk 1]
```diff
@@ -272,17 +275,14 @@ <h2>Bliss OS comes in a few flavors</h2>
...
 		explanation of our currently supported variants to help things along.</p>
                 <div class="row mt-4">
-                    <div class="col-sm p-2 mx-2 bg-primary">
-                        <h2 class="my-3">Bliss OS 11.x</h2>
-                        <p>These are generic builds based on Android 9 (Pie) and BlissROMs sources. This is considered
-                        our current stable branch.</p>
+                    <div class="col-sm p-2 mx-2 bg-primary rounded">
+                        <h2 class="my-3">Bliss OS 14.10.x</h2>
+                        <p>These are generic builds based on Android 11 and BlissROMs sources. These builds are considered experimental.</p>
                         <a class="btn btn-secondary mb-3 mx-1" target="blank" href="https://forum.xda-developers.com/t/bliss-os-pie-for-pcs-lts.3855917/">XDA thread</a>
```
Comment (Xtr126) [inline index.html:281]: Wrong XDA thread link for Bliss OS 14.10, line 281

Comment (hmtheboy154) [reply index.html:281]: @SGNight oh yea, can you fix it alongside with the new banner and screenshots ?

**Pull request merged**